corehq/apps/domain/views.py

```diff
@@ -664,6 +664,16 @@ def plan(self):
                         'renew_url': reverse(SubscriptionRenewalView.urlname, args=[self.domain]),
                     })
 
+        if subscription:
+            credit_lines = CreditLine.get_credits_by_subscription_and_features(subscription)
+            has_credits_in_non_general_credit_line = any(
```

Comment:
@nickpell This is the core of 'has credits in non general credit lines'. Can you see if it looks good?

Also per the comment on spec, should I set this to True if no credit-lines exist?